README.md

```diff
@@ -25,7 +25,7 @@
     <a href="https://zenodo.org/badge/latestdoi/250213286"><img src="https://zenodo.org/badge/250213286.svg" alt="DOI"></a>
 </p>
 
-`🤗Datasets` is a lightweight library providing **two** main features:
+`🤗 Datasets` is a lightweight library providing **two** main features:
```

Comment:
IMHO, here and everywhere else in this doc needs to be either 🤗 `datasets` or 🤗 Datasets - no backticks!

backticks are for the module name, which is not `🤗 Datasets` but `datasets`

Reply:
since there is only one python package called `datasets` if you go with the former, you probably don't need the emoji.

I'd say to be consistent with `transformers` docs, it should be: 🤗 Datasets (as in 🤗 Transformers).

Reply:
Unless you are writing: 

> Main differences between `🤗 Datasets` and `tfds`

a few paras below, which looks very unbalanced.

Here you probably want to go for one of:
1.  `datasets` and `tfds`
2.  🤗 Datasets and TensorFlow `tfds`

Just a suggestion of course...